scripts/create-relocatable-package-python3.py

```diff
@@ -228,19 +243,24 @@ def generate_file_list(executables):
 
 file_list = generate_file_list(dependencies(packages))
 ar = tarfile.open(args.output, mode='w|gz')
+# relocatable package format version = 2
+with open('build/.relocatable_package_version', 'w') as f:
+    f.write('2\n')
+ar.add('build/.relocatable_package_version', arcname='.relocatable_package_version')
```

Comment:
You can add content directly, without creating a file. See TarFile.addfile() and io.BytesIO(). We already use it scylla-python3: 

```python
    ti = tarfile.TarInfo(name=os.path.join("bin", pybin))
    ti.size = len(thunk)
    ti.mode = 0o755
    ar.addfile(ti, fileobj=io.BytesIO(thunk))
```